Mailbox2.ino

```diff
@@ -176,4 +188,11 @@ void loop() {
   // TODO - add temperatur sensor and humitidy sensor, send data via MQTT.
   // Because everybody needs to know what the temperature is in their mailbox.
 
+  // Temperature reading
+  const float temp = float(digitalRead(tempSensorInPin)); // interpret input bits as float
```

Comment:
I think this would probably not work on its own. Look at http://playground.arduino.cc/Main/DHTLib